benchmark/bin/evaluate_benchmark.rs

```diff
@@ -0,0 +1,214 @@
+use odra::{DeployReport, GasReport};
+use std::ops::{Div, Mul};
+use std::path::PathBuf;
+use std::process::exit;
+use std::{env, fs};
+
+pub fn main() {
+    let args: Vec<String> = env::args().collect();
+    if args.len() < 3 {
+        println!(
+            "Usage: {} <current_gas_report_path> <base_gas_report_path>",
+            args[0]
+        );
+        exit(1);
+    }
+
+    println!("Evaluating benchmark...");
+    let current_gas_report_path = PathBuf::from(&args[1]);
+    let base_gas_report_path = PathBuf::from(&args[2]);
+    let current_gas_report = load_gas_report(&current_gas_report_path);
+    let base_gas_report = load_gas_report(&base_gas_report_path);
+
+    exit(compare_gas_reports(&current_gas_report, &base_gas_report));
+}
+
+fn load_gas_report(file: &PathBuf) -> GasReport {
+    let gas_report_json =
+        fs::read(file).unwrap_or_else(|_| panic!("Failed to read file: {:?}", file));
+    serde_json::from_str(
+        String::from_utf8(gas_report_json)
+            .unwrap_or_else(|_| panic!("Failed to parse file {:?}", file))
+            .as_str()
+    )
+    .unwrap_or_else(|_| panic!("Failed to parse json: {:?}", file))
+}
+
+fn compare_gas_reports(current_gas_report: &GasReport, base_gas_report: &GasReport) -> i32 {
+    let mut exit_code = 0;
```

Comment:
What's the difference between exit code 1,2,3?

Reply:
1 - differences in gas
2 - errors
3 - both

Reply:
It is now not utilised, but may be in the future